doc/workshop/exercises/analysis/denoising/denoising.rst

```diff
@@ -74,13 +74,13 @@ another :ref:`range <ranges>` to keep only points that are within the range
 :math:`-100 <= Z <= 3000`.
 
 Both :ref:`ranges <ranges>` are passed as a comma-separated list to the
```

Comment:
This paragraph needs a rework. No more ranges now that we are using `filters.expression`